samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -120,13 +126,15 @@
         LOG.error("Received exception from in JobCoordinator Processing!", throwable);
         stop();
       });
+    Map<String, SystemAdmin> systemAdminMap = new JavaSystemConfig(config).getSystemAdmins();
+    systemAdmins = new SystemAdmins(Util.javaMapAsScalaMap(systemAdminMap));
```

Comment:
If most usages of this Map constructor parameter are in Java, maybe cleaner to make it a java.util.Map instead?

Reply:
Good point. I have made the change as suggested.